tools/patch/tensorflow/cc/saved_model/BUILD

```diff
@@ -0,0 +1,12 @@
+diff --git a/tensorflow/cc/saved_model/BUILD b/tensorflow/cc/saved_model/BUILD
```

Comment:
In earlier review I missed that you didn't modify Dockerfile to actually apply this patch... so I don't think it is actually needed...

Reply:
Correct. Removed the patch.